lua/nvim-treesitter/install.lua

```diff
@@ -361,10 +367,22 @@ local function install_lang(lang, ask_reinstall, cache_folder, install_folder, w
       return
     end
 
-    local yesno = fn.input(lang .. " parser already available: would you like to reinstall ? y/n: ")
-    print "\n "
-    if not string.match(yesno, "^y.*") then
-      return
+    -- Now check if the ABI is not correct
+    local parser = inspect_lang(lang)
+    if parser._abi_version == vim.treesitter.language_version then
```

Comment:
do we want an exact match or are older compatible versions also ok? I think the runtime known it's minimal compatible version.

Reply:
It indeed knows it, but I am not sure this has been propagated to lua yet
If not I am going to make a PR for that.